miso-web/src/main/webapp/scripts/arrayRun_ajax.js

```diff
@@ -0,0 +1,275 @@
+(function(ArrayRun, $, undefined) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Do not use "undefined" to declare a parameter - use another name. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=javascript%3AS2137)